integration-tests/utils/integration-helpers.js

```diff
@@ -200,8 +200,8 @@ const clickNavChildAndLoad = async (page, tab) => {
 }
 
 const viewWorkspaceDashboard = async (page, token, workspaceName) => {
-  await click(page, clickable({ textContains: 'View Workspaces' }))
   await signIntoTerra(page, { token })
+  await click(page, clickable({ textContains: 'View Workspaces' }))
```

Comment:
Was there a reason to switch this order?

Reply:
See my comment for this change.